localize/comments/15/NuGet.Core.resources.dll.lci

```diff
@@ -1,459 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We're not using the file right now. 
NuGet.Core.dll is part of the NuGet.exe 
And the resources in NuGet.Core.dll will need to be localized when we localize the exe. 
There's a plan to localize NuGet.exe and I assume it'll be via microbuild. 

If you've checked that all the resources for nuget.core.dll are included in the nuget.exe resources then you can remove. 

And yes, it was used when we referenced nuget.core in the VSIX.